models/Receipt.js

```diff
@@ -1,12 +1,31 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 const User = require("./User");
+const types = require('./types')
 
 const receiptSchema = new Schema({
-  tenant: { type: Schema.ObjectId, ref: "User" },
-  payment_date: Date,
-  payment_due: Date,
-  price_paid: Number // Change to Decimal128 if Decimal128 datatype provides floats (it should)
+  tenant: { 
+    type: Schema.ObjectId, 
+    ref: "User" 
+  },
+  receipt_date_created: { //Date when receipt is created after splitting.
+    type: Date,
+    default: Date.now,
+    required: true
+  },
+  payment_date: {
+    type: Date
+  },
+  payment_due: {
+    type: Date,
+    required: true
+  },
+  price_paid: Decimal128, 
```

Comment:
Please write a comment here @vsutinah so that we are aware about its data type as you described above.

Reply:
Ok, commented.